src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1733,26 +1733,34 @@ Listing the Catalog for existing Buckets is done using the following HTTP reques
 
     GET http://localhost:8080/catalog/buckets
 
-It is also possible to list all Workflows inside a particular Bucket using its `bucket_id`:
+It is also possible to list all Workflows inside a particular Bucket using its `bucketName`:
 
-    GET http://localhost:8080/catalog/buckets/{bucket_id}/resources
+    GET http://localhost:8080/catalog/buckets/{bucketName}/resources
 
 Then, we can fetch a particular Workflow based on its `name`:
 
-    GET http://localhost:8080/catalog/buckets/{bucket_id}/resources/{name}
+    GET http://localhost:8080/catalog/buckets/{bucketName}/resources/{name}
 
 NOTE: By default, the Catalog will always return the latest version of the Workflows.
 
+=== Bucket naming requirements
+
+Every bucket is identified by unique bucket name. The bucket naming should follow the following rules:
+
+* The bucket name can be between 3 and 63 characters long, and can contain only lower-case characters, numbers, and dashes.
```

Comment:
The bucket name must be between 3 and 63 characters long and can contain only lower-case characters, numbers, and dashes.

Reply:
numbers and dashes (no comma before the last and)